src/webfaf/templates/problems/problems_table.html

```diff
@@ -0,0 +1,95 @@
+{% macro problems_table(problems) %}
+    {% if problems %}
+      <noscript><p>WARNING!! This page is using JavaScript to filter and sort the table.</p></noscript>
+      <table class="table datatable table-striped table-bordered table-condensed">
+        <thead>
+          <tr>
+            <th>ID</th>
+            <th>Component</th>
+            <th>Count</th>
```

Comment:
Does the change of order of the columns 
from 
`ID | Component | Crash function | State | External bugs | Probably fixed | Count `
to
`ID | Component | Count | Crash function | State | Bugs | Probably fixed `
help something? 

I would keep it as it is.